apps/dashboard/src/components/workflow-editor/step-utils.ts

```diff
@@ -48,3 +57,11 @@ export const updateStepInWorkflow = (
     }),
   };
 };
+
+export const createStep = (type: StepTypeEnum): Step => ({
+  name: STEP_NAME_BY_TYPE[type],
+  stepId: '',
+  slug: '_st_',
```

Comment:
Does the slug start with `_`? Shouldn't this be? 
```suggestion
  slug: 'st_',
```

Reply:
Should be correct, its `delay-step_st_<id>`

i just copied it to different file so I guess it worked before